### PR TITLE
Feature: Create config file on deploy

### PR DESCRIPTION
**Pull request by GregTheGreek**
This pr adds the ability for a configuration file to be create when  is called that conforms to the structure expected by cfgbuilder

### CODE REVIEW

File: cb-sol-cli/yarn.lock [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (steviezhang) [inline cb-sol-cli/yarn.lock:2]: Is this in sync with our `package-lock.json`?

Comment (GregTheGreek) [reply cb-sol-cli/yarn.lock:2]: oh shoot i use yarn , i can remove this. 

pacakge-lock.json directly pulls from package.json anyways.

**Comment by ansermino** [comment]
Please update documentation

File: cb-sol-cli/cmd/deploy.js [hunk 2]
```diff
@@ -18,6 +18,8 @@ const deployCmd = new Command("deploy")
     .option('--erc20', 'Deploy erc20 contract')
     .option('--erc721', 'Deploy erc721 contract')
     .option('--centAsset', 'Deploy centrifuge asset contract')
+    .option('--config', 'Creates a configuration file based on the deployment', false)
+    .option('--configFile <path>', 'Path to store configuration file to', './config.json')
```
Comment (ansermino) [inline cb-sol-cli/cmd/deploy.js:22]: Thought we removed this?

File: cb-sol-cli/README.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ The root command (`cb-sol-cli`) has some options:
 --url <value>                 URL to connect to
 --gasLimit <value>            Gas limit for transactions 
 --gasPrice <value>            Gas limit for transactions 
+--networkId <value>	      Network id
```
Comment (ansermino) [inline cb-sol-cli/README.md:19]: This isn't a thing..?

**Review by ansermino — APPROVED** [review]

**Pull request merged**